docs/debugging.md

```diff
@@ -56,3 +56,9 @@ You *can't*:
 * Perform most other ordinary debugging scenarios.
 
 Development of further debugging scenarios is an on-going focus of the engineering team.
+
+## Troubleshooting tips
+
+If you are running into errors, the following tip may help.
+
+* In the debugger tab, open the developer tools in your browser, and in the console, execute localStorage.clear(). This will remove any breakpoints that were set.
```

Comment:
Let's drop the bullet for now and revise the passage to the following to adhere to our style guidelines. The following is a cut-and-paste [and I assume it's "debugger" not "Debugger" (uppercase)] ...

> In the \*\*debugger\*\* tab, open the developer tools in your browser. In the console, execute \`localStorage.clear()\` to remove any breakpoints.